hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -198,6 +201,28 @@ public void testCreateTableTxnBuilder() throws Exception {
     }
   }
 
+  @Test
+  public void testLoadTableWithCustomMetricReporter() throws Exception {
+    Schema schema = getTestSchema();
+    TableIdentifier tableIdent = TableIdentifier.of(DB_NAME, "tbl");
+    String location = temp.newFolder("tbl").toString();
+
+    HiveCatalog catalog = new HiveCatalog();
+    catalog.initialize(
+        "hive", ImmutableMap.of("metrics-reporter-impl", TestMetricsReporter.class.getName()));
+    try {
+      Transaction txn =
+          catalog.buildTable(tableIdent, schema).withLocation(location).createTransaction();
```

Comment:
Is it necessary to create the table through a transaction? Isn't this the result of copy-pasting from the thest above that tests transactional create table? :)

Reply:
I don't think, its necessary to use transaction. Let me change it.